cookiecutter/main.py

```diff
@@ -41,6 +41,7 @@ def cookiecutter(
     :param config_file: User configuration file path.
     :param default_config: Use default values rather than a config file.
     :param password: The password to use when extracting the repository.
+    :param directory: Subdirectory within repo where cookiecutter.json lives.
```

Comment:
Something more like: A relative path to a cookiecutter template in the cloned repository?